From 1a1f74d0b65ac97982dbcfe394455723ff9646f2 Mon Sep 17 00:00:00 2001
From: "Tekriwal, Tanuj" <tanuj.tekriwal@intel.corp-partner.google.com>
Date: Mon, 22 Jun 2026 08:52:25 +0000
Subject: [PATCH] Fixing error with naming in kernel

This will fix the error:
"kernel/android_ack/common/tools/testing/selftests/
devices/probe/boards/Dell", needed by "out/target/
product/caas/obj/kernel/.config.check", missing and
no known rule to make it

Tracked-On: NA
Signed-off-by: Tekriwal, Tanuj <tanuj.tekriwal@intel.corp-partner.google.com>
---
 groups/kernel/AndroidBoard.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/groups/kernel/AndroidBoard.mk b/groups/kernel/AndroidBoard.mk
index e513654..b502bbe 100644
--- a/groups/kernel/AndroidBoard.mk
+++ b/groups/kernel/AndroidBoard.mk
@@ -123,7 +123,7 @@ KERNEL_CONFIG_DEPS = $(strip $(KERNEL_DEFCONFIG) $(KERNEL_DIFFCONFIG))
 CHECK_CONFIG_SCRIPT := $(LOCAL_KERNEL_SRC)/scripts/diffconfig
 CHECK_CONFIG_LOG :=  $(LOCAL_KERNEL_PATH)/.config.check
 
-KERNEL_DEPS := $(shell find $(LOCAL_KERNEL_SRC) \( -name *.git -prune \) -o -print )
+KERNEL_DEPS := $(shell find $(LOCAL_KERNEL_SRC) \( -name *.git -prune \) -o \( -path "*/tools/testing/selftests*" -prune \) -o -print )
 
 KERNEL_MAKE_CMD:= \
       PATH="$(PWD)/prebuilts/build-tools/linux-x86/bin:$(TARGET_KERNEL_CLANG_PATH):$(PWD)/prebuilts/gcc/linux-x86/host/x86_64-linux-glibc2.17-4.8/x86_64-linux/bin:$$PATH" \
-- 
2.34.1

